src/Reflection/ReflectionNamedType.php

```diff
@@ -54,6 +61,48 @@ public function isBuiltin(): bool
         return array_key_exists(strtolower($this->name), self::BUILT_IN_TYPES);
     }
 
+    public function getClass(): ReflectionClass
+    {
+        if (! $this->isBuiltin()) {
+            return $this->reflector->reflectClass($this->name);
+        }
+
+        if (
+            $this->owner instanceof ReflectionEnum
```

Comment:
Shouldn't an enum have a reflectable type? 

After all, it is still a class-alike symbol: https://3v4l.org/dsl4L#v8.1rc3

Reply:
The owner is `ReflectionMethod` in your example.

`ReflectionEnum` is owner here:

```php
enum Foo: int {}
```

Backing type can currently contain only `int` or `string`.